lms/djangoapps/lms_xblock/test/test_runtime.py

```diff
@@ -39,12 +39,25 @@ def test_escaped(self, test_string):
         self.assertNotIn('/', quote_slashes(test_string))
 
 
+class BlockMock(Mock):
```

Comment:
I can add that. I made this class just to get existing tests to pass. Doing `.__class__` introspection on a Mock gives you `None` for these attributes, so I needed to create a subclass to declare the handlers so it would still see them in the runtime tests.